test/fixtures/files/merged_pull_request.json

```diff
@@ -1,332 +1,66 @@
+"comments_url": "https://api.github.com/repos/octobox/octobox/issues/403/comments",
+"events_url": "https://api.github.com/repos/octobox/octobox/issues/403/events",
+"html_url": "https://github.com/octobox/octobox/pull/403",
+"id": 245148325,
+"number": 403,
+"title": "Align unread counts with the selected filter",
+"user": {
+"login": "chrisarcand",
+"id": 2430490,
+"avatar_url": "https://avatars2.githubusercontent.com/u/2430490?v=4",
+"gravatar_id": "",
+"url": "https://api.github.com/users/chrisarcand",
+"html_url": "https://github.com/chrisarcand",
+"followers_url": "https://api.github.com/users/chrisarcand/followers",
+"following_url": "https://api.github.com/users/chrisarcand/following{/other_user}",
+"gists_url": "https://api.github.com/users/chrisarcand/gists{/gist_id}",
+"starred_url": "https://api.github.com/users/chrisarcand/starred{/owner}{/repo}",
+"subscriptions_url": "https://api.github.com/users/chrisarcand/subscriptions",
+"organizations_url": "https://api.github.com/users/chrisarcand/orgs",
+"repos_url": "https://api.github.com/users/chrisarcand/repos",
+"events_url": "https://api.github.com/users/chrisarcand/events{/privacy}",
+"received_events_url": "https://api.github.com/users/chrisarcand/received_events",
+"type": "User",
+"site_admin": false
+},
+"labels": [],
+"state": "closed",
+"locked": false,
+"assignee": null,
+"assignees": [],
+"milestone": null,
+"comments": 0,
+"created_at": "2017-07-24T17:12:34Z",
+"updated_at": "2017-07-25T15:20:00Z",
+"closed_at": "2017-07-24T18:17:30Z",
+"pull_request": {
+"url": "https://api.github.com/repos/octobox/octobox/pulls/403",
+"html_url": "https://github.com/octobox/octobox/pull/403",
+"diff_url": "https://github.com/octobox/octobox/pull/403.diff",
+"patch_url": "https://github.com/octobox/octobox/pull/403.patch"
+},
+"body": "Before:\n\n![](http://screenshots.chrisarcand.com/jm18s.gif)\n\nAfter:\n\n![](http://screenshots.chrisarcand.com/xu17y.gif)",
+"closed_by": {
+"login": "chrisarcand",
+"id": 2430490,
+"avatar_url": "https://avatars2.githubusercontent.com/u/2430490?v=4",
+"gravatar_id": "",
+"url": "https://api.github.com/users/chrisarcand",
+"html_url": "https://github.com/chrisarcand",
+"followers_url": "https://api.github.com/users/chrisarcand/followers",
+"following_url": "https://api.github.com/users/chrisarcand/following{/other_user}",
+"gists_url": "https://api.github.com/users/chrisarcand/gists{/gist_id}",
+"starred_url": "https://api.github.com/users/chrisarcand/starred{/owner}{/repo}",
+"subscriptions_url": "https://api.github.com/users/chrisarcand/subscriptions",
+"organizations_url": "https://api.github.com/users/chrisarcand/orgs",
+"repos_url": "https://api.github.com/users/chrisarcand/repos",
+"events_url": "https://api.github.com/users/chrisarcand/events{/privacy}",
+"received_events_url": "https://api.github.com/users/chrisarcand/received_events",
+"type": "User",
+"site_admin": false
+}
```

Comment:
??? Why is this diff different? Was the original incorrect? (I think I just stole it from the API docs and not an actual reqest)

Reply:
Don't worry, that's me messing with things, mostly trying to get the pr data from the issues endpoint so that includes labels 🙈

Reply:
No worries, just curiosity! 💙 👍